_data/projects/auth-jwt.yml

```
@@ -0,0 +1,13 @@
+---
+name: auth-jwt
+desc: 'A demo to learn JWT by reverse engineering'
```

' is not needed here.
```suggestion
desc: A demo to learn JWT by reverse engineering
```

---

Fixed in latest commit. Can you please review again.

---

Ohk